addon/mixins/dom.js

```diff
@@ -84,13 +84,6 @@ function removeHandlerFromListenerData(handler) {
  @public
  */
 export default Mixin.create({
-  init() {
-    this._super(...arguments);
-
-    this._listeners = [];
```

Comment:
Sorry, I know I mentioned to remove the `init` but I forgot that we need to shape the object properly also. 

Can you bring this section back, but instead of assigning these to a new empty array we need to assign to undefined.